test/src/touchscreen.spec.ts

```diff
@@ -26,29 +26,41 @@ describe('Touchscreen', function () {
   setupTestBrowserHooks();
   setupTestPageAndContextHooks();
 
-  it('should tap the button', async () => {
+  it('should report touches', async () => {
     const {page, server} = getTestState();
     const iPhone = KnownDevices['iPhone 6']!;
     await page.emulate(iPhone);
-    await page.goto(server.PREFIX + '/input/button.html');
-    await page.tap('button');
+    await page.goto(server.PREFIX + '/input/touches.html');
+    const button = (await page.$('button'))!;
+    await button.tap();
```

Comment:
let's not change the existing test. Please add a new test for the ElementHandle.tap

Reply:
ok, I reAdd the previous one